packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -508,6 +509,20 @@ const fn = new lambda.Function(this, 'MyFunction', {
 });
 ```
 
+And if needed, one might utilize an `sns.Topic` instead of `sqs.Queue` as dead-letter queue:
+
+```ts
+import * as sns from '@aws-cdk/aws-sns';
+
+const dlq = new sns.Topic(this, 'DLQ');
+const fn = new lambda.Function(this, 'MyFunction', {
+  runtime: lambda.Runtime.NODEJS_12_X,
+  handler: 'index.handler',
+  code: lambda.Code.fromInline('exports.handler = function(event, ctx, cb) { return cb(null, "hi"); }'),
+  deadLetterQueue: dlq,
```

Comment:
This needs to be updated to a `deadLetterTopic` right. Also the variable name needs updating too